doc/user_manual/metrics.tex

```diff
@@ -66,24 +66,25 @@ \section{Metrics}
     \xmlString{dice}, \xmlString{hamming}, \xmlString{yule} and so on.
   \item \textbf{Pairwise Metric}, compute the distance or kernel between each pair of the two collections of input
     or observations in n-dimensional space.
-    \nb These metrics can be only used in the clustering post-processor of data mining. 
+    \nb These metrics can be only used in the clustering post-processor of data mining.
   \item \textbf{Other metric}, such as \xmlString{DTW}.
 \end{itemize}
 
-The valid \textbf{MetricID}s are: \xmlNode{SKL}, \xmlNode{ScipyMetric}, \xmlNode{DTW}, \xmlNode{CDFAreaDifference},
-and \xmlNode{PDFCommonArea}. This XML node requires the following attributes:
+The valid \textbf{MetricID}s are: \xmlAttr{SKL}, \xmlAttr{ScipyMetric}, \xmlAttr{DTW}, \xmlAttr{CDFAreaDifference},
+and \xmlAttr{PDFCommonArea}. This XML node requires the following attributes:
 \begin{itemize}
   \item \xmlAttr{name}, \xmlDesc{required string attribute}, user-defined name of this metric. \nb As with other
     objects, this name can be used to refer to this specific entity from other input blocks in the XML.
+  \item \xmlAttr{subType}, \xmlDesc{required string attribute}, the desired type of Metric to use.
 \end{itemize}
 
-\nb If you are using \xmlNode{ScipyMetric}, please pay more attention on the weight associated with the metric
+\nb If you are using \xmlAttr{ScipyMetric}, please pay more attention on the weight associated with the metric
 calculations. Scipy does not normalize the weight during the calculation, and the results can be significant difference
 from the normalized weight.
 
-In RAVEN, lots of metrics are just interfaces directly with metrics available within \textbf{Scipy} and
+In RAVEN, lots of metrics are interfaces directly coupled with metrics available within \textbf{Scipy} and
 \textbf{SciKit-Learn}. In this case, the algorithm for the metrics is choosen by the subnode \xmlNode{metricType}
-under the parent node \xmlNode{SKL} (metric from SciKit-Learn) or \xmlNode{ScipyMetric} (metric from Scipy). For
+under the parent node \xmlAttr{SKL} (metric from SciKit-Learn) or \xmlAttr{ScipyMetric} (metric from Scipy). For
```

Comment:
"under the parent node \xmlAttr{SKL}" --> "under the parent node with subType equal to \xmlAttr{SKL}"?

Reply:
Yes, you're right. Fixing this.